configuration/configuration_reference.md

```diff
@@ -877,6 +857,211 @@ can be used to **customize how config repo plugin works when parsing this specif
 
 [top](#top)
 
+## &lt;svn&gt; {#config-repo-svn}
+
+The config repo `<svn>` material element specifies the location of your code base in Subversion repository.
+
+### Attributes
+
+| Attribute | Required | Description |
+|-----------|----------|-------------|
+| url | Yes | URL for the remote repository, for example: 'http://www.thoughtworks-studios.com/go-agile-release-management/svn/myproject/trunk'. Go supports the following protocols for subversion: http, https, svn and svn+ssh, but does not support 'file:///'. |
```

Comment:
Lets not use that url - I believe it doesn't exist. I guess you copied it from the exiting svn description, so you will have to change that one too.